Sources/Rendering/Core/ColorTransferFunction/index.d.ts

```diff
@@ -201,6 +202,14 @@ export interface vtkColorTransferFunction extends vtkObject {
     withAlpha: boolean
   ): Float32Array;
 
+  /**
+   * Construct a color transfer function from a vtkDataArray.
+   * @param {vtkDataArray} array
+   */
```

Comment:
Would be nice to document how tuple size is used by the function

Reply:
The documentation has been moved here

Reply:
good catch!